Makefile

```diff
@@ -37,3 +37,7 @@ check-crds: crds ## Check crd
 	@git diff --quiet || \
     (echo "Need to update crds, please run 'make crds'"; \
 	exit 1)
+
+.PHONY: update-greptimedb-version
```

Comment:
if you use the `version` argument, you can run the makefile command like:

```
make update-greptimedb-version version=<version>
```

Reply:
refactor to https://github.com/GreptimeTeam/helm-charts/pull/118/commits/92510dd171d9a6c19bef9598f1fbc0126ed06dcc